docs/serving/configuring-autoscaling.md

```diff
@@ -245,6 +245,17 @@ spec:
         - image: gcr.io/knative-samples/helloworld-go
 ```
 {{< /tab >}}
+{{% tab name="Global (ConfigMap)" %}}
```

Comment:
Can you add a "Global (Operator)" too? :)

Reply:
OKay :)